app/models/npq_profile.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class NpqProfile < ApplicationRecord
+  belongs_to :user
+  belongs_to :npq_lead_provider
+  belongs_to :npq_course
```

Comment:
Hm, I wonder if `npq_course` should have `npq_lead_provider` in it, and whether `belongs_to :npq_lead_provider` should be `through`. Thoughts?

Reply:
My understanding is all providers offer all the courses. You don't select a provider first then get a provider specific course list.